website/docs/tech-explanation/quantum-resistance.md

```diff
@@ -0,0 +1,108 @@
+
+- **Other blockchains:** Act like a pocket calculator. They calculate specific math (ECC) very fast, but you cannot add new buttons.
+- **Nervos CKB:** Acts like a generic CPU. If you need a new mathematical operation (like the matrix multiplications used in Lattice-based cryptography), you simply upload the code for it.
+
+This abstraction turns Quantum Resistance from a "Governance Crisis" into a simple "User Choice." Users can choose to migrate to quantum-resistant cryptography without a hard fork. By ten years later, if there are quantum computers that can break the current cryptography, CKB users can follow the process for upgrades in the following:
+
+1.  **Deploy:** Developers deploy a new Lock Script implementing PQC algorithms (e.g., SPHINCS+).
+2.  **Adopt:** Users create new addresses referencing this script's `Code Hash`.
+3.  **Migrate:** Users transfer assets from old addresses to new quantum-secure ones.
+
+The network does not fork. Old addresses continue to function. The upgrade is permissionless and gradual.
+
+### The Data Size Limitation
+
+Even if other blockchains successfully coordinate a hard fork, they still face a physical limit: **Data Size**. Quantum-resistant signatures are orders of magnitude larger than classical ones.
+
+- **Secp256k1 (Bitcoin/ETH):** ~64 bytes
+- **ML-DSA (Dilithium):** ~2.5 KB (~40x larger)
+- **SPHINCS+:** 8 KB – 49 KB (~125x - 760x larger)
+
+Take Bitcoin as an [example](https://x.com/bensig/status/1985426927893823667), a simple consolidation of 100 UTXOs using ML-DSA signatures could result in a **250KB+ transaction**, potentially costing hundreds of thousands of dollars in fees during peak congestion.
+
+CKB doesn't face such a problem, since the verification of signatures is done by separting the rule(Lock Script) from the proof(Witness). There are also mechanisms for grouping the same Lock Scripts so multiple inputs can share one signature in the transaction Witnesses. That makes moving 100 Cells with SPHINCS+ signatures as simple as moving 1 Cell with SPHINCS+ signature. You can learn more about this in [Script Group Execution](/docs/tech-explanation/script-group-exe) and [Witness](/docs/tech-explanation/witness).
+
+## What CKB has achieved
+
+### Implementation Details: SPHINCS+ on CKB
+
+In 2023, The CKB team and Cryptape researchers have successfully implemented a production-ready **Quantum Resistant Lock Script** using **SPHINCS+**. In August 2024, NIST has finally approved SPHINCS+ as a quantum-resistant digital signature algorithm in its FIPS 205 standard. In 2025, The CKB team started to deploy SPHINCS+ Lock Script on the CKB mainnet.
+
+- **Repository:** [cryptape/quantum-resistant-lock-script](https://github.com/cryptape/quantum-resistant-lock-script)
+
```

Comment:
Suggest to add note we finished security audit partner with ScaleBit.
After they uploaded the report to their offcial website, we can add the report link here.